### PR TITLE
:white_check_mark: added tests for `starknet_getNonce`

### PR DESCRIPTION
**Pull request by Trantorian1**
:+1: Tests include the following:

- **invalid block**: client should return error.
- **invalid contract**: client should return error.
- **ERC721 contract**: client should return nonce of 0.
- **ERC20 contract**: client should return nonce of 0.
- **account contract**: client should return *non-0* nonce.
- **account proxy contract**: client should return *non-0* nonce.

### CODE REVIEW

File: unit_tests/src/constants.rs [hunk 1]
```diff
@@ -6,5 +6,70 @@ pub const STARKGATE_ETH_CONTRACT_ADDR: &str = "0x049d36570d4e46f48e99674bd3fcc84
 pub const INVALID_CONTRACT_ADDR: &str = "0x4269DEADBEEF";
 pub const CONTRACT_ADDR: &str = "0x03a20d4f7b4229e7c4863dab158b4d076d7f454b893d90a62011882dc4caca2a";
 pub const CONTRACT_KEY: &str = "0x00f920571b9f85bdd92a867cfdc73319d0f8836f0e69e06e4c5566b6203f75cc";
+
+/**
+ * Contract address for StarkGate ETH starknet bridge.
```
Comment (antiyro) [inline unit_tests/src/constants.rs:11]: Same here please use rustdoc official nomination ///

Comment (Trantorian1) [reply unit_tests/src/constants.rs]: Updated all comments in commit cb063efab7adad0d47d778a6ddcfb453dae72e36 and 2a4c95f3d9f6708715b6f8d34d1486ee42eaf2c0

File: .vscode/settings.json [hunk 2]
```diff
@@ -4,12 +4,15 @@
...
-        "pathfinder"
+        "pathfinder",
+        "get_nonce"
     ],
     "cSpell.words": [
         "Deoxys",
         "rstest",
         "serde",
-        "starknet"
+        "STARKGATE",
```
Comment (antiyro) [inline .vscode/settings.json:14]: Please git ignore .vscode

Comment (Trantorian1) [reply .vscode/settings.json:14]: On it!

Comment (Trantorian1) [reply .vscode/settings.json]: Removed `.vscode` in commit ef7a5ac864d56ec7819cdf02f61ec85c822d10b5

**Pull request merged**